src/components/Greeting.js

```diff
@@ -0,0 +1,21 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchGreeting, selectGreeting } from './redux/greetingSlice';
+
+function Greeting() {
+  const greeting = useSelector(selectGreeting);
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchGreeting());
+  }, [dispatch]);
+
+  return (
+    <div>
+      <h1>{greeting}</h1>
+      <p>fefef</p>
+    </div>
```

Comment:
- [ ] According to the project requirements, the React app uses the url and displays a random greeting, however, I can't see any random treating message ⬇️  from your seed. Kindly address this 👍🏻 

![Screenshot from 2022-10-27 05-47-31](https://user-images.githubusercontent.com/79058364/198187012-b74d22a8-3add-493b-8bc0-bbb7ab9472b5.png)